modules/upgrading-oadp10-to12-in-mtc.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/upgrading-3-4.adoc
+// * migration_toolkit_for_containers/upgrading-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-upgrading-oadp-for-mtc-18_{context}"]
+= Upgrading OADP 1.0 to 1.2 for {mtc-full} 1.8.0
+
+
+.Procedure
+
+
+* For each subscription, patch the OADP operator from OADP 1.0 to OADP 1.2, by issuing:
++
+[source,terminal]
+----
+$ oc -n openshift-migration patch subscription redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace --type merge --patch '{"spec": {"channel":"stable-1.2"}}'
+----
++
+[NOTE] 
+==== 
+Sections indicating the user-specific returned `NAME` values that are used for the installation of MTC & OADP, respectively. 
+While it will be similar, it is unlikely that the returned value will be `redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace`, which is used in this example.
+====
++
+Which will return the following:
++
+[source,terminal]
+----
+subscription.operators.coreos.com/redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace patched
+----
+
++
+[NOTE]
+====
+* If `installPlanApproval` is set to `Automatic`, Operator Lifecycle Manager (OLM) will begin the upgrade process.
+* If `installPlanApproval` is set to `Manual`, you will need to approve each `installPlan` before OLM will begin the upgrades.
+====
+
```

Comment:
* "Sections indicating the user-specific returned `NAME` values that are used for the installation of MTC & OADP, respectively." This sentence is incomplete, not sure what it is conveying, or if it needs to be a note.
* Suggest moving the statement about the example value to a note following the output. 
* The existing note after the output should probably not be a note block. 

Suggest something like this:

```suggestion
[source,terminal]
----
$ oc -n openshift-migration patch subscription redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace --type merge --patch '{"spec": {"channel":"stable-1.2"}}'
----
+
[NOTE] 
==== 
Sections indicating the user-specific returned `NAME` values that are used for the installation of MTC & OADP, respectively. 
====
+
.Example output
[source,terminal]
----
subscription.operators.coreos.com/redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace patched
----
+
[NOTE]
====
The returned value will be similar to `redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace`, which is used in this example.
====
+
--
* If the `installPlanApproval` parameter is set to `Automatic`, the Operator Lifecycle Manager (OLM) begins the upgrade process. 
* If the `installPlanApproval` parameter is set to `Manual`, you must approve each `installPlan` before the OLM begins the upgrades.
--
```